app/Domain/Integrations/Organizer.php

```diff
@@ -11,7 +11,7 @@
     public function __construct(
         public UuidInterface $id,
         public UuidInterface $integrationId,
-        public UuidInterface $organizerId
+        public string $organizerId
```

Comment:
I wonder if the migration should also be updated for clarity? I think it still works because all the faulty UUId's have 35 instead of 36 characters? It's an open question, don't know for sure myself.

Reply:
We can switch it to a standard text column but I feel I want to keep the hope we will be able to revert to actual UUIDs down the line. For now yes the `uuid` Blueprint method effectively just  creates a `char(36)` which is fine for our use case so not sure either 😬